cmds/cli/main.go

```diff
@@ -0,0 +1,176 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+
+	Boot "github.com/u-root/u-root/pkg/boot"
```

Comment:
User local case import name:

```suggestion
	"github.com/u-root/u-root/pkg/boot"
```